packages/env-loader/src/index.ts

```diff
@@ -8,53 +8,36 @@ export type EnvVars = {
   [key: string]: string
 }
 
-const getAllEnvVars = async (scriptName: string): Promise<EnvVars> => {
-  // CLI OPTIONS
-  const cliOptions = getCliOptions(scriptName)
+export const processEnv = async (command: string): Promise<void> => {
+  // CLI
+  const { args: cliArgs, options: cliOptions } = getCliOptionsAndArgs()
 
-  // ENV FILE VARS
+  // ENV FILE
   const envVars = getEnvFileVars(process.env.APP_ENV)
 
-  // For now, don't fetch CMS feature flags for tests.
-  // Will fail CI.
+  // CMS FEATURE FLAGS
+  let cmsFeatureFlags
   if (process.env.APP_ENV === 'test') {
-    return {
-      ...envVars,
-      ...cliOptions,
-    }
+    // For now, don't fetch CMS feature flags for tests. Will fail CI.
+    cmsFeatureFlags = {}
+  } else {
+    const drupalBaseUrlProp = 'NEXT_PUBLIC_DRUPAL_BASE_URL'
+    const drupalBaseUrl =
+      cliOptions[drupalBaseUrlProp] || envVars[drupalBaseUrlProp]
+    cmsFeatureFlags = await getCmsFeatureFlags(drupalBaseUrl)
   }
 
-  // CMS FEATURE FLAGS
-  const drupalBaseUrlProp = 'NEXT_PUBLIC_DRUPAL_BASE_URL'
-  const drupalBaseUrl =
-    cliOptions[drupalBaseUrlProp] || envVars[drupalBaseUrlProp]
-  const cmsFeatureFlags = await getCmsFeatureFlags(drupalBaseUrl)
-
-  // Return all options with proper cascading:
-  // 1. CLI options have highest precedence
-  // 2. CMS feature flags next
-  // 3. ENV file vars last
```

Comment:
I think this comment (or something to that effect) is valuable to keep even if it is technically self-documented in the code